ext/storage_sources/local_store/local_store.c

```diff
@@ -593,10 +594,17 @@ local_file_copy(LOCAL_STORAGE *local, WT_SESSION *session, const char *src_path,
     WT_FILE_SYSTEM *wt_fs;
     wt_off_t copy_size, file_size, left;
     ssize_t pos;
+    size_t pathlen;
     int ret, t_ret;
-    char buffer[1024 * 64];
+    char buffer[1024 * 64], *tmp_path;
 
     dest = src = NULL;
+    pathlen = strlen(dest_path) + 10;
+    if ((tmp_path = malloc(pathlen)) != NULL)
+        if (snprintf(tmp_path, pathlen, "%s.TEMP", dest_path) >= (int)pathlen) {
```

Comment:
Minor nit: Can you use `TMP` instead of TEMP for consistency? All other temporary files use some form of `TMP` including the library and test/format.